pynamodb/exceptions.py

```diff
@@ -134,6 +134,7 @@ class CancellationReason:
     """
     code: str
     message: Optional[str] = None
+    item: Optional[Dict[str, Dict[str, Any]]] = None
```

Comment:
Maybe we should call it `raw_item` given that typically in PynamoDB we also attempt to map the item?

Reply:
makes sense